src/modules/achievements/AchievementSortOptions.ts

```diff
@@ -0,0 +1,27 @@
+export enum AchievementSortOptions {
+    default = 0,
+    progress = 1,
+}
+
+export type AchievementSortOptionConfig = {
+    // Display text in sort dropdown
+    text: string;
+
+    // How to get the comparison value from an Achievement
+    getValue: (a) => any;
+
+    // true if the default sort direction should be descending
+    invert?: boolean;
+};
+
+export const AchievementSortOptionConfigs: Record<AchievementSortOptions, AchievementSortOptionConfig> = {
+    [AchievementSortOptions.default]: {
+        text: 'Default',
+        getValue: (a) => a.bonusWeight,
+    },
```

Comment:
This doesn't seem to be our default sorting option.

Could you change this to be how they are currently sorted and add Bonus % as its own option.